logsapi/server.go

```diff
@@ -19,10 +20,30 @@ type LogMessage struct {
 	Record interface{} `json:"record"`
 }
 
+var (
+	// set up logging defaults for our own logging output
+	log = logrus.WithFields(logrus.Fields{
+		"source": "hny-lambda-ext-logsapi",
+	})
+)
+
+func init() {
+	envLogLevel, ok := os.LookupEnv("HNY_LOG_LEVEL")
+	if !ok {
+		envLogLevel = "info"
+	}
+	parsedLogLevel, err := logrus.ParseLevel(envLogLevel)
+	if err != nil {
+		parsedLogLevel = logrus.InfoLevel
+	}
+	logrus.SetLevel(parsedLogLevel)
+}
+
 // handler receives batches of log messages from the Lambda Logs API. Each
 // LogMessage is sent to Honeycomb as a separate event.
 func handler(libhoneyClient *libhoney.Client) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
+		log.Debug("handler - log batch received")
```

Comment:
Added this to the logs handler for when someone would like to see logs getting received and events getting queued.